test/evergreen.yml

```diff
@@ -5559,7 +5568,6 @@ buildvariants:
 - <<: *mac_test_template
   name: macos-14-arm64
   display_name: "macOS 14 (ARM64)"
-  patch_only: true
   run_on:
     - macos-14-arm64
   #batchtime: 120 # 2 hours
```

Comment:
We commented out `batchtime` in the original ticket on the off chance it overrode the `patch_only` rule. Please re-enable it.